client/layout/sidebar/style.scss

```diff
@@ -144,12 +144,13 @@
 
 		// External indicator for sections that aren't available in Calypso
 		&.gridicons-external {
+			height: 18px;
+			left: auto;
+			margin: 0;
 			position: absolute;
 			right: 15px;
-			left: auto;
+			top: auto;
```

Comment:
Without `top:auto` CSS from `.external-link` component was leaking in otherwise. (`top: 2px`)